pkg/services/rendering/interface.go

```diff
@@ -35,5 +35,6 @@ type Service interface {
 	IsAvailable() bool
 	Render(ctx context.Context, opts Opts) (*RenderResult, error)
 	RenderErrorImage(error error) (*RenderResult, error)
+	RenderUnavailableImage() *RenderResult
```

Comment:
Not sure I this is good decision. What if `Render` returned the fake image if there is no render service instead?

Reply:
@bergquist please explain why you think it's not a good decision?

Reply:
I guess @bergquist means it would be less complex if we could avoid adding another interface method?

Reply:
@marefr sorry. I should definitely describe why I think so. :heart: 

Large interfaces makes code brittle. The smaller the better. But the main reason I think we can consider moving this into `Render` is that this code now enforces the caller to check if there is a render service or not and make decisions based on that. So it offloads the responsibility to the caller rather than solving the problem itself. 

If the render service was responsible for returning a mock image and logging about it, other areas of the codebase would not have to care about it. 

But don't threat this as a blocker for merging the code. If you want to merge and roll forward that's fine.

Reply:
Thanks. In alerting wanted to log a warning as well, but I see now that I can do that in rendering package instead :+1: